### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by hodovani** [BerniWittmann/vue-unused-components-checker#1]
I tried to use the lib and it says, that every component that I have is unused.

Could you please help me.

What kind of information should I provide to give a better picture of the issue?

**Comment by BerniWittmann** [comment]
Hey, 
thanks for letting me know. 
Could you provide an example how you import your components?

**Comment by hodovani** [comment]
Thanks @BerniWittmann 
Here is an example:
```javascript
import AppHeader from '../../molecules/layout/app-header';
...
import Table from '../../components/atoms/table/table';
```

**Comment by BerniWittmann** [comment]
Thanks :)
I'm having a suspicion
I created the library for a project of mine and just made it public. Therefore, this library is pretty much tailored to the specifics of this project. 
Let me check whether I can improve the detection to be more versatile to other import urls 

Thanks for raising this & I'll let you know

**Comment by BerniWittmann** [comment]
Hey @hodovani 
I just released a new version of this package (v1.1.0) that should fix this and i also improved some other things to make it more robust.
Please let me know whether this solves your issue :)
